pkg/handler/handler.go

```diff
@@ -96,26 +100,82 @@ type ReadmeUtgaveFilter struct {
 	First  *int32
 }
 
-func (r *resolver) ReadmeUtgaver(filter *ReadmeUtgaveFilter) ([]ReadmeUtgave, error) {
-	if filter == nil {
-		return r.readmes, nil
-	}
-	var filteredReadmes []ReadmeUtgave
-	for _, r := range r.readmes {
-		if filter.Year != nil && *filter.Year != r.Year {
-			continue
+func getLink(name string) string {
+	return fmt.Sprintf("https://storage.googleapis.com/%v/%v", bucketName, name)
+}
+
+func getReadmes(ctx context.Context, client storage.Client, query string) ([]ReadmeUtgave, error) {
+	pdfQuery := &storage.Query{Prefix: "pdf/" + query}
+	imageQuery := &storage.Query{Prefix: "images/" + query}
+
+	re := regexp.MustCompile(`(?P<Year>\d{4})-(?P<Utgave>\d{2})`)
```

Comment:
Pls add some tests for this fancy-shmancy regex :tm:

Reply:
That is not a bad idea :robot: